AlembicImporter/Packages/com.unity.formats.alembic/Runtime/Scripts/Exporter/AlembicCameraParams.cs

```diff
@@ -1,6 +1,6 @@
 using UnityEngine;
 
-namespace UTJ.Alembic
+namespace UnityEngine.Formats.Alembic.Util
 {
```

Comment:
Typically the Util namespace is used for things that are required from both the importer and the exporter